mapflow/mapflow.py

```diff
@@ -3140,6 +3179,40 @@ def update_processing(self):
         dialog.setup(processing)
         dialog.deleteLater()
 
+    def check_project_sharing(self):
+        if len(self.dlg.projectsCombo) != 0:
```

Comment:
1. Too much inlayed if-s may look cumbersome. There are ways to reduce the layers:
```
# allows to exit immediately after "bad" condition
if len(self.dlg.projectsCombo) == 0:
    return
...
```
2. Can we take len() of  a widget, don't we need to call items() or something?

Reply:
2. - Yes, len() returns a number of projects, which works for checking if projectsCombo is not empty (in case someting is wrong with default project)